seeder/src/main/interfaces/shiinobi.ts

```diff
@@ -30,8 +30,8 @@ class Shiinobi {
 		}
 	}
 
-	#spawn({ command, id }: { command: _COMMANDS; id?: number }) {
-		let _command: string[] = [command];
+	#spawn({ command, id }: { command: _COMMANDS; id?: number }): Promise<unknown> {
+		const _command: string[] = [command];
```

Comment:
@baseplate-admin do you agree with this change?
well this is eslint auto fixed one

Reply:
well, this is not something we need.

Adding implicit return types